sync-core/src/test/java/com/cloudant/sync/replication/ReplicationTestBase.java

```diff
@@ -37,6 +37,8 @@
     protected CouchClientWrapper remoteDb = null;
     protected CouchClient couchClient = null;
 
+    private long randomDBSuffix = System.currentTimeMillis();
```

Comment:
This isn't a random suffix right now. Why not just actually generate a random suffix rather than relying on the current system time?